deploy/complete/terraform/mushop.tf

```diff
@@ -19,7 +19,7 @@ resource "helm_release" "mushop" {
 
   set {
     name  = "global.mock.service"
-    value = var.mushop_mock_mode_all ? "all" : "false"
+    value = var.mushop_mock_mode_all ? "all" : "none"
```

Comment:
👍

Reply:
Yeah, Terraform 0.14 force the "false" to be boolean and test for the type.